xdsl/interactive/app.py

```diff
@@ -23,13 +17,12 @@
 from textual.widgets.text_area import TextAreaTheme
 
 from xdsl.dialects.builtin import ModuleOp
+from xdsl.interactive._pasteboard import pyclip_copy
```

Comment:
Maybe this will work? I think it's unhappy with the underscore, which implies that it should not be imported from outside this scope.

```suggestion
from ._pasteboard import pyclip_copy
```